src/babel-preset/node.js

```diff
@@ -10,15 +10,16 @@ module.exports = {
           // version 8 and babel will stop transpiling async/await
           // because they are supported in the "current" version of node
           node: 'current',
+          include: ['babel-plugin-for-await']
```

Comment:
Need to make sure this is includes in the `package.json`

Reply:
Good catch!